src/scripts/main.js

```diff
@@ -21,7 +21,26 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  function buildList(obj) {
+    const ul = document.createElement('ul');
+
+    for (const key in obj) {
+      const li = document.createElement('li');
+      li.textContent = key;
+
+
+      if (Object.keys(obj[key]).length > 0) {
+        li.appendChild(buildList(obj[key]));
+      }
+
+      ul.appendChild(li);
+    }
+
+    return ul;
+  }
+
+  element.innerHTML = '';
+  element.appendChild(buildList(data));
```

Comment:
You are clearing the entire innerHTML of the element each time you call `createTree`. This can be inefficient if the element contains a lot of content. Consider clearing the content in a more efficient way, for instance by removing child nodes.